wallet/common/symbol/src/utils/transaction-from-symbol.js

```diff
@@ -0,0 +1,603 @@
+	const baseTransaction = baseTransactionFromSymbol(transaction, config);
+
+	switch (transaction.type.value) {
+	case TransactionType.AGGREGATE_BONDED:
+	case TransactionType.AGGREGATE_COMPLETE:
+		return aggregateTransactionFromSymbol(transaction, config);
+	case TransactionType.TRANSFER:
+		return transferTransactionFromSymbol(transaction, config);
+	case TransactionType.ADDRESS_ALIAS:
+		return addressAliasTransactionFromSymbol(transaction, config);
+	case TransactionType.MOSAIC_ALIAS:
+		return mosaicAliasTransactionFromSymbol(transaction, config);
+	case TransactionType.NAMESPACE_REGISTRATION:
+		return namespaceRegistrationTransactionFromSymbol(transaction, config);
+	case TransactionType.MOSAIC_DEFINITION:
+		return mosaicDefinitionTransactionFromSymbol(transaction, config);
+	case TransactionType.MOSAIC_SUPPLY_CHANGE:
+		return mosaicSupplyChangeTransactionFromSymbol(transaction, config);
+	case TransactionType.MOSAIC_SUPPLY_REVOCATION:
+		return mosaicSupplyRevocationTransactionFromSymbol(transaction, config);
+	case TransactionType.SECRET_LOCK:
+		return secretLockTransactionFromSymbol(transaction, config);
+	case TransactionType.HASH_LOCK:
+		return hashLockTransactionFromSymbol(transaction, config);
+	case TransactionType.SECRET_PROOF:
+		return secretProofTransactionFromSymbol(transaction, config);
+	case TransactionType.VRF_KEY_LINK:
+		return vrfKeyLinkTransactionFromSymbol(transaction, config);
+	case TransactionType.ACCOUNT_KEY_LINK:
+		return accountKeyLinkTransactionFromSymbol(transaction, config);
+	case TransactionType.NODE_KEY_LINK:
+		return nodeKeyLinkTransactionFromSymbol(transaction, config);
+	case TransactionType.VOTING_KEY_LINK:
+		return votingKeyLinkTransactionFromSymbol(transaction, config);
+	case TransactionType.MOSAIC_GLOBAL_RESTRICTION:
+		return mosaicGlobalRestrictionTransactionFromSymbol(transaction, config);
+	case TransactionType.MOSAIC_ADDRESS_RESTRICTION:
+		return mosaicAddressRestrictionTransactionFromSymbol(transaction, config);
+	case TransactionType.ACCOUNT_OPERATION_RESTRICTION:
+		return accountOperationRestrictionTransactionFromSymbol(transaction, config);
+	case TransactionType.ACCOUNT_ADDRESS_RESTRICTION:
+		return accountAddressRestrictionTransactionFromSymbol(transaction, config);
+	case TransactionType.ACCOUNT_MOSAIC_RESTRICTION:
+		return accountMosaicRestrictionTransactionFromSymbol(transaction, config);
+	case TransactionType.MULTISIG_ACCOUNT_MODIFICATION:
+		return multisigAccountModificationTransactionFromSymbol(transaction, config);
+	case TransactionType.ACCOUNT_METADATA:
+		return accountMetadataTransactionFromSymbol(transaction, config);
+	case TransactionType.NAMESPACE_METADATA:
+		return namespaceMetadataTransactionFromSymbol(transaction, config);
+	case TransactionType.MOSAIC_METADATA:
+		return mosaicMetadataTransactionFromSymbol(transaction, config);
+	}
+
+	return baseTransaction;
+};
+
+export const baseTransactionFromSymbol = (transaction, config) => {
```

Comment:
missing function docs..same below

Reply:
Removed export since it is internal private helper.